nginx/webapp.conf

```diff
@@ -0,0 +1,16 @@
+server {
+    listen       80;
+    listen  [::]:80;
+    server_name  localhost;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri /index.html;
```

Comment:
We need to add this ruleto make it work with react-router https://stackoverflow.com/a/43954597

Reply:
Could you elaborate why we need this additional config?

Reply:
Are you already testing the production environment? This seems like something I would find or even thought about at this moment. I am really impressed